tests/lib/prepare.sh

```diff
@@ -365,15 +365,6 @@ prepare_classic() {
 
         disable_refreshes
 
-        echo "Ensure that the bootloader environment output does not contain any of the snap_* variables on classic"
```

Comment:
could you skip this just on s390x ? I think we should still run this on at least amd64 and the arm platforms since those use bootloaders for which this check makes sense

Reply:
thanks for the review, sure.